base/deprecated.jl

```diff
@@ -409,12 +395,7 @@ for (f,t) in ((:int,    Int), (:int8,   Int8), (:int16,  Int16), (:int32,  Int32
               (:int64,  Int64), (:int128, Int128), (:uint,   UInt), (:uint8,  UInt8),
               (:uint16, UInt16), (:uint32, UInt32), (:uint64, UInt64), (:uint128,UInt128))
```

Comment:
These should be handled by https://github.com/JuliaLang/julia/blob/sb/parsefloat/base/deprecated.jl#L312-L317